option optimize_for = LITE_RUNTIME;

message ActivityRewardRequest
{
 required uint32 act_id = 1;
 required uint32 task_id = 2;
}
